src/android/toga_android/window.py

```diff
@@ -48,6 +48,9 @@ def set_size(self, size):
     def set_app(self, app):
         pass
 
+    def set_full_screen(self, is_full_screen):
```

Comment:
The concept of "full screen" doesn't really mean anything on mobile or web - apps are always full screen (or, at least, full viewport). We should flag this as an API endpoint that doesn't exist on Android, iOS and django/web - if you check the dummy backend's app.py, you can see some examples of backend-specific annotation.